UCSLibraries/src/main/java/it/cnr/iit/utility/Utility.java

```diff
@@ -158,6 +138,27 @@ public static boolean createPathIfNotExists( File file ) {
         return true;
     }
 
+    /*public static String readFileAsString( String resource ) throws URISyntaxException, IOException {
+        ClassLoader classLoader = Utility.class.getClassLoader();
+        Path path = Paths.get( classLoader.getResource( resource ).toURI() );
+
+        StringBuilder sb = new StringBuilder();
+        try (BufferedReader br = new BufferedReader( new FileReader( path.toString() ) )) {
+            for( String line; ( line = br.readLine() ) != null; ) {
+                sb.append( line );
+            }
+        }
+        return sb.toString();
+    }*/
```

Comment:
why dont you just delete the above commented out code block?

Reply:
oh yeah, it's a leftover :+1: